msvc/scripts/tests/run-jit.bat

```diff
@@ -0,0 +1,87 @@
+@ECHO OFF
+
+SET TEMP_PATH=%PATH%
+SET PATH=
+SET MONO_RESULT=1
+
+CALL setup-env.bat
+IF NOT ERRORLEVEL == 0 (
+	ECHO Failed to setup mono paths.
+	GOTO ON_ERROR
+)
+
+CALL setup-toolchain.bat
```

Comment:
Instead of errorlevel,  you can do the  short circuit idiom that is popular in other languages. like call foo || ( error stuff )